src/modules/pos/validator/components/SentStakes/SentStakes.js

```diff
@@ -33,10 +34,20 @@ function ClaimRewardsDialogButton({ address }) {
   const hasClaimableRewards =
     rewardsClaimable?.data?.length &&
     rewardsClaimable?.data?.reduce((acc, curr) => BigInt(curr.reward) + acc, BigInt(0)) > BigInt(0);
+  const {
+    appEvents: {
+      transactions: { rewards },
+    },
+  } = useContext(ApplicationBootstrapContext);
+  const notification = rewards.length && BigInt(rewards[0]?.reward || 0) > BigInt(0);
+  const notificationStyles = notification ? styles.notification : '';
 
   return (
     <DialogLink component="claimRewardsView">
-      <SecondaryButton disabled={!hasClaimableRewards}>{t('Claim rewards')}</SecondaryButton>
+      <SecondaryButton disabled={!hasClaimableRewards}>
+        {t('Claim rewards')}
+        <span className={notificationStyles} />
```

Comment:
I think it would be nice to have a component called `badge` whose job would be to show notifications so we don't need to duplicate style in certain use case

Reply:
Fixed